internal/configs/virtualserver.go

```diff
@@ -511,9 +511,32 @@ func generatePath(path string) string {
 	return path
 }
 
+func generateReturnBlock(text string, code int, defaultCode int) *version2.Return {
```

Comment:
I strongly suggest not using `block` with `return` in the code. The code  in this package deals with NGINX configuration. Better to be consistent with the terminology when possible.

Reply:
This doesn't make any sense. Go code != nginx config. In templates it makes sense since is kind of a mix between the two worlds, here it doesn't. But thanks for your suggestion.

Reply:
> Go code != nginx config.
this code is in the package config that deals with NGINX config.

generateReturnBlock - is generating something that doesn't exist in NGINX. I suggest sticking to NGINX terminology, `generateReturnBlock` -> `generateReturn`

Note that using ReturnBlock in `generateLocationForReturnBlock` would be OK, because here ReturnBlock is something we refer in YAML file, not in the NGINX config.

Reply:
Inconsistent terminology adds complexity. This will make code less maintainable. The code of our project is growing and it should be a priority to keep the codebase maintainable.